packages/react/src/formElements/MdMultiSelect.tsx

```diff
@@ -11,37 +9,36 @@ import MdChevronIcon from '../icons/MdChevronIcon';
 import MdClickOutsideWrapper from '../utils/MdClickOutsideWrapper';
 import MdCheckbox from './MdCheckbox';
 
-export interface MdMultiSelectOptionProps {
-  text: string | number;
-  value: string | number;
+export interface MdMultiSelectOption {
+  text: string;
+  value: string;
 }
 
 export interface MdMultiSelectProps {
   label?: string | null;
-  options?: MdMultiSelectOptionProps[];
-  selected?: MdMultiSelectOptionProps[];
+  options?: MdMultiSelectOption[];
+  // 3.0.0: Rename previous prop selected to selectedOptions
+  selectedOptions?: MdMultiSelectOption[];
   placeholder?: string;
   disabled?: boolean;
-  /**
-   * Replaces previous 'size'-prop for reducing overall width of whole component from large to either medium or small.
-   */
   mode?: 'large' | 'medium' | 'small';
   helpText?: string;
   error?: boolean;
   errorText?: string;
   showChips?: boolean;
   closeOnSelect?: boolean;
   id?: string;
-  onChange?(_e: React.ChangeEvent): void;
   dropdownHeight?: number;
+  // 3.0.0: Rename previous prop onChange to onSelectOption and use MdMultiSelectOption as parameter rather than event
+  onSelectOption?(_option: MdMultiSelectOption): void;
```

Comment:
Kan være en ulempe å ikke få tak i eventet fra utsiden. Typisk case ville vært om man hadde behov for å kjøre event.stopPropagation() i en event handler, men mulig det er et teoretisk behov 🤷

Reply:
Ja, var litt usikker her. Men generelt i md-komp så sendes ut en egen type "Option" når det ikke følger en default html-komponent. Det var ganske mye krøll i imlementasjonen og me dtyping osv når man bruker changeevent, og siden det brukes andre steder tenkte jeg det var lette generelt å bruke options som standard